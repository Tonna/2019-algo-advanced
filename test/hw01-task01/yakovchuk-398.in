25
1 1 1 3 2 3 5 2 1 1 7 8 5 1 4 13 12 17 10 2 3 20 18 4
5
5 4
13 9
1 7
17 10
23 8
