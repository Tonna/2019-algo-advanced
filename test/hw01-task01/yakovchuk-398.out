6
25
11
29
44
